exporter/opentelemetry-exporter-jaeger-proto-grpc/README.rst

```diff
@@ -5,6 +5,8 @@ OpenTelemetry Jaeger Protobuf Exporter
     Since v1.35, the Jaeger supports OTLP natively. Please use the OTLP exporter instead.
     Support for this exporter will end July 2023.
 
+    This package is no longer being tested.
```

Comment:
Do we even need to add this note with the message of ending support above?

Reply:
I don't think it hurts to mention no more testing is being done for this package.